crates/biome_js_analyze/src/lint/style/use_consistent_curly_braces.rs

```diff
@@ -419,3 +428,12 @@ fn contains_single_space(node: &JsxExpressionAttributeValue) -> bool {
         )
     })
 }
+
+fn contains_forbidden_chars(str_literal: &JsStringLiteralExpression) -> bool {
+    let forbidden_chars = ['>', '"', '\'', '}'];
```

Comment:
You should move this array inside a `const`, so we don't create this array every time we call this function